packages/react-components/react-nav-preview/src/components/NavGroup/NavGroup.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { useNavGroup } from './useNavGroup';
+import { renderNavGroup } from './renderNavGroup';
+import type { NavGroupProps } from './NavGroup.types';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useNavGroupStyles } from './useNavGroup.styles';
+// import { useCustomStyleHook_unstable } from '@fluentui/react-shared-contexts';
+
+/**
+ * A tab provides a selectable item in a tab list.
+ */
+export const NavGroup: ForwardRefComponent<NavGroupProps> = React.forwardRef((props, ref) => {
```

Comment:
I find Nav vs NavGroup naming a bit confusing.  My initial take is that a NavGroup would contain a Nav similar to how a TabList contains a Tab.  Will you have a NavItem someday? Is there a suffix to add to Nav that would help describe it as a container (NavPanel, NavTree, NavAccordion, etc.)?

Reply:
correct - I'm still noodling on that. Ideally, the Nav will live in or be a Drawer.

We're still flushing out spec, and working with our ContentDesigner on naming. So for now we should operate under the assumption that *all* names are subject to drastically changing until the spec is completely nailed down, probably sometime in the late February/March timeframe.